src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -37,9 +37,9 @@
 
   <PropertyGroup>
       <VersionFeature21>30</VersionFeature21>
-      <VersionFeature31>22</VersionFeature31>
-      <VersionFeature50>14</VersionFeature50>
-      <VersionFeature60>2</VersionFeature60>
+      <VersionFeature31>23</VersionFeature31>
+      <VersionFeature50>15</VersionFeature50>
+      <VersionFeature60>3</VersionFeature60>
```

Comment:
@marcpopMSFT am I correct that another PR bumping these numbers won't cause conflicts❔  I didn't look for one.